plugin/geoip/README.md

```diff
@@ -16,35 +18,54 @@ import (
 // ...
 if getLongitude := metadata.ValueFunc(ctx, "geoip/longitude"); getLongitude != nil {
     if longitude, err := strconv.ParseFloat(getLongitude(), 64); err == nil {
-		// Do something useful with longitude.
-	}
+        // Do something useful with longitude.
+    }
 } else {
     // The metadata label geoip/longitude for some reason, was not set.
 }
 // ...
 ```
 
 ## Databases
+
 The supported databases use city schema such as `City` and `Enterprise`. Other databases types with different schemas are not supported yet.
 
 You can download a [free and public City database](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data).
 
 ## Syntax
-```txt
+
+```text
 geoip [DBFILE]
 ```
+
+or
+
+```text
+geoip [DBFILE] {
+    edns-subnet-processing
+}
+```
+
+By default, geoip plugin doesn't honor EDNS0 subnet extension, but it can be enabled using the extended syntax:
+
 * **DBFILE** the mmdb database file path.
+* `edns-subnet-processing` (optional): by default, geoip plugin read client IP and ignores [EDNS0 subnet](https://en.wikipedia.org/wiki/EDNS_Client_Subnet) extension. Using this option enables reading source IP from EDNS0 subnet when available. This help identifying closest source IP address as well as making GeoIP test processing easy: `dig +subnet=1.2.3.4 @dns-server.example.com www.geo-aware.com`.
```

Comment:
```suggestion
* `edns-subnet-processing`  Optional. For the geo IP address, use [EDNS0 subnet](https://en.wikipedia.org/wiki/EDNS_Client_Subnet) if available instead of the source IP of the DNS request. This helps with identifying the closest source IP address as well as making GeoIP testing easy: `dig +subnet=1.2.3.4 @dns-server.example.com www.geo-aware.com`.
```

It's not clear what you mean by "This helps with identifying the closest source IP address".

Reply:
@chrisohaver I'm theorycrafting here, but in theory, public recursors can forward original requestors' IP addresses in an EDNS0 subnet attachment. We just open the gates for such a behavior.

Otherwise, good changes.